server-side/tools/nginx/default

```diff
@@ -0,0 +1,38 @@
+# /etc/nginx/sites-available/default
+# Default nginx site config
+# /etc/nginx/sites-enabled/default links to this file
+
+upstream ultra-gro_web {
+    server localhost:8000;
+}
+server {
+    listen 80;
+    server_name sales.ultragro.net;
+    return 301 https://$server_name$request_uri;
+}
+server {
+    listen 443;
+    server_name sales.ultragro.net;
+
+    ssl on;
+    ssl_certificate /etc/letsencrypt/live/sales.ultragro.net/fullchain.pem;
```

Comment:
I would remove the specific names of these files and replace them with variables.

Reply:
Done